common/components/AddressFieldFactory/AddressFieldDropdown.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { AppState } from 'reducers';
+import translate, { translateRaw } from 'translations';
+import { setCurrentTo, TSetCurrentTo } from 'actions/transaction';
+import { getLabelAddresses } from 'selectors/addressBook';
+import { getToRaw } from 'selectors/transaction/fields';
+import { Address, Identicon } from 'components/ui';
+import './AddressFieldDropdown.scss';
+
+interface StateProps {
+  labelAddresses: ReturnType<typeof getLabelAddresses>;
+  currentTo: ReturnType<typeof getToRaw>;
+}
+
+interface DispatchProps {
+  setCurrentTo: TSetCurrentTo;
+}
+
+type Props = StateProps & DispatchProps;
+
+interface State {
+  activeIndex: number | null;
+}
+
+class AddressFieldDropdown extends React.Component<Props> {
+  public state: State = {
+    activeIndex: null
+  };
+
+  public componentDidMount() {
+    window.addEventListener('keydown', this.handleKeyDown);
+  }
+
+  public componentWillUnmount() {
+    window.removeEventListener('keydown', this.handleKeyDown);
+  }
+
+  public render() {
+    const { currentTo } = this.props;
+    const noMatchContent = currentTo.startsWith('0x') ? null : (
+      <li className="AddressFieldDropdown-dropdown-item AddressFieldDropdown-dropdown-item-no-match">
+        <i className="fa fa-warning" /> {translate('NO_LABEL_FOUND_CONTAINING')}"{currentTo}".
```

Comment:
Should be a space between translate and quotes.

Reply:
For context, I think that some of the translations have trailing spaces, but I don't want to rely on that in case someone forgets to add one when doing the translations in other languages. Better to do it once for all cases.

Reply:
:+1: